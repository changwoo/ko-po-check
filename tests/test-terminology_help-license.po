# -*-text-*-
# test
#
msgid ""
msgstr ""
"Project-Id-Version: test\n"
"POT-Creation-Date: 2014-10-11 00:00+0900\n"
"PO-Revision-Date: 2014-10-11 00:00+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: C/legal.xml:4
msgid "Creative Commons Attribution-Share Alike 3.0 United States License"
msgstr "Creative Commons Attribution-Share Alike 3.0 United States License"

#: C/legal.xml:4
msgid "CreativeCommons Attribution-ShareAlike 3.0 Unported License"
msgstr "CreativeCommons Attribution-ShareAlike 3.0 Unported License"
