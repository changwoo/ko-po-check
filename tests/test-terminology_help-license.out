-:17: 일관성을 위해 다음과 같이 번역합니다: 크리에이티브 커먼즈 저작자표시-동일조건변경허락 3.0 미합중국 라이선스
-:21: 일관성을 위해 다음과 같이 번역합니다: 크리에이티브 커먼즈 저작자표시-동일조건변경허락 3.0 Unported 라이선스
